data.go

```diff
@@ -48,6 +49,9 @@ type Data interface {
 	// as this one (shallow comparison)
 	Equal(other Data) bool
 
+	// Compare compares given data to this data, row by row.
```

Comment:
```suggestion
	// Compare compares given data to this data, row by row
```
no `.` in this interface on methods level comments :)